packages/cli/src/commands/generate.js

```diff
@@ -75,6 +77,7 @@ export default {
 
     // Forcing static target anyway
     config.target = TARGETS.static
+    consola.info(`Enforcing ${chalk.bold.blue('static')} target because of \`nuxt generate\` command`)
```

Comment:
Maybe also changing it to an advice to use `target: 'static'` as it will be default behavior?

Reply:
Probably also with a short-link to https://nuxtjs.org/blog/going-full-static? :D